diff --git a/node_modules/react-native-bottom-tabs/react-native-bottom-tabs.podspec b/node_modules/react-native-bottom-tabs/react-native-bottom-tabs.podspec
index d4b9289..025b6eb 100644
--- a/node_modules/react-native-bottom-tabs/react-native-bottom-tabs.podspec
+++ b/node_modules/react-native-bottom-tabs/react-native-bottom-tabs.podspec
@@ -25,7 +25,10 @@ Pod::Spec.new do |s|
   s.dependency 'SDWebImageSVGCoder', '>= 1.7.0'
 
   s.pod_target_xcconfig = {
-    'DEFINES_MODULE' => 'YES'
+    'USE_HEADERMAP' => 'YES',
+    'DEFINES_MODULE' => 'YES',
+    'CLANG_CXX_LANGUAGE_STANDARD' => 'c++20',
+    'SWIFT_COMPILATION_MODE' => 'wholemodule',
   }
 
   install_modules_dependencies(s)
